generated/nidaqmx/system/watchdog.py

```diff
@@ -254,7 +252,7 @@ def expir_trig_trig_on_network_conn_loss(self):
         check_for_error(error_code)
 
     @property
-    def expir_trig_trig_type(self):
+    def expir_trig_type(self):
```

Comment:
I like the way we handle triggers for normal tasks, with the trigger object for each type: https://github.com/ni/nidaqmx-python/blob/master/src/nidaqmx/_task_modules/triggers.py. however, that is a bit overkill for this since there is only one option as far as I can tell: digital. Time is an option, but I don't think there is a way to configure it, so that's cool.

Anyways, the analog for this in a normal trigger is: `triggers.arm_start_trigger.trig_type`. Since we don't have an object, I don't think we need to repeat `trig`, so I like this: `expiration_trigger_type`